en/components/date_time_editor.md

```diff
@@ -0,0 +1,173 @@
+</igx-input-group>
+```
+
+<div class="sample-container loading" style="height:100px">
+    <iframe id="datetime-basic-iframe" data-src="{environment:demosBaseUrl}/scheduling/datetime-basic" width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="datetime-basic-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+
+### Usage
+To get started with the Ignite UI for Angular Date Time Directive, let's first import the `IgxDateTimeEditorModule` in the **app.module.ts** file:
+
+```typescript
+// app.module.ts
+
+...
+import { IgxDateTimeEditorModule } from 'igniteui-angular';
+
+@NgModule({
+    ...
+    imports: [..., IgxDateTimeEditorModule ],
+    ...
+})
+export class AppModule {}
+```
+
+
+To use an input as a date time editor, set an igxDateTimeEditor directive and a valid date object as value. In order to have complete editor look and feel, wrap the input in an [input group](input_group.md). This will allow you to not only take advantage of the following directives [`igxInput`]({environment:angularApiUrl}/classes/igxinputdirective.html), [`igxLabel`]({environment:angularApiUrl}/classes/igxlabeldirective.html), `igx-prefix`, `igx-suffix`, [`igx-hint`]({environment:angularApiUrl}/classes/igxhintdirective.html), but cover common scenarios when dealing with form inputs as well.
+
+#### Binding 
+A basic configuration scenario setting a Date object as a [`value`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#value):
+```typescript
+public date = new Date();
+```
+
+```html
+<igx-input-group>
+    <input type="text" igxInput igxDateTimeEditor [value]="date"/>
+</igx-input-group>
+```
+
+To create a two-way data-binding, set an ngModel:
+```html
+<igx-input-group>
+    <input type="text" igxInput igxDateTimeEditor [(ngModel)]="date"/>
+</igx-input-group>
+```
+<div class="divider--half"></div>
+
+
+#### Keyboard Navigation
+Date Time Editor Directive has intuitive keyboard navigation that makes it easy to jump trough different DateParts, increment, decrement, etc. without having to touch the mouse.
+
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>Arrow Left</kbd> / <kbd>Right</kbd> - navigates between date sections. Jumps from current section to the same caret position of the next/previous section. On start/end sections, first jump to the other end of the same section following the direction of the arrow.
+
+- <kbd>Arrow Up</kbd> / <kbd>Down</kbd> - increment/decrement date portions. See related [`isSpinLoop`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#isspinloop)
+
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>;</kbd> - set the current day and time in the editor via a shortcut (as it is in Excel).
+
+
+### Examples
+
+#### Display and input format
```

Comment:
Can we have some additional information about input formats. If I am using date editor for the first time I should realize at first place that setting `[igxDateTimeEditor]="'dd/MM/yyyy'" sets the mask and in the same time this is the input fromat. It will be nice if this is described. Also do we have all supported display formats somewhere listed? Link to this here will be great.

Reply:
I will add more details as links to wiki pages containing more info - like * [`DateTime-Editor`](https://github.com/IgniteUI/igniteui-angular/wiki/DateTime-Editor-Specification) are not common in the topics.